pkg/scorecard_result.go

```diff
@@ -107,7 +107,7 @@ func (r *ScorecardResult) AsString(showDetails bool, logLevel zapcore.Level, wri
 	}
 
 	table := tablewriter.NewWriter(os.Stdout)
-	header := []string{"Score", "Reason", "Name"}
+	header := []string{"Score", "Name", "Reason"}
```

Comment:
Should this be in order -> Name, Score, Reason?

Reply:
@olivekl only asked to swap reason and name. @olivekl can you confirm?

Reply:
I'd suggested just the swap, but either way is fine. With the scores first, users can skim down the first column to get a quick overview of how they're doing. With the names first, the tests are more obvious.